clearblade.md

```diff
@@ -1323,6 +1323,59 @@ This function does not return query results.
 	db.exec("delete from traffic where entrance='Store_Entrance';");
 ~~~~
 
+## ClearBlade.Database.performOperation(callback, argument)
```

Comment:
ClearBlade.Database() also has a update.
Since it takes a parameter as well. So there are like two types of declarations which are possible